### PR TITLE
WebAPI: Update constructor pages (part 8) 

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Continuing #14001
### Summary

Updates parameter lists.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by wbamberg — DISMISSED** [review]
Thanks @OnkarRuikar !

File: files/en-us/web/api/blob/blob/index.md [hunk 1]
```diff
@@ -43,10 +43,6 @@ new Blob(array, options);
         characters into the blob without changing them. To convert newlines to the host
         system's native convention, specify the value `native`.
 
-### Return value
```
Comment (teoli2003) [inline files/en-us/web/api/blob/blob/index.md:46]: Why removing these? They are in the template.

Comment (OnkarRuikar) [reply files/en-us/web/api/blob/blob/index.md:46]: These are constructor pages and not method pages. Why state the obvious?

**Comment by OnkarRuikar** [comment]
I've removed the surrounding backticks from the macros and reverted the return value sections.

**Review by teoli2003 — APPROVED** [review]
This looks really good! Thank you!

**Pull request merged**